tests/protocol/addingMargin/test_deposit_collareral_using_TestToken.py

```diff
@@ -17,14 +17,20 @@ def test_deposit_collateral(sovryn,set_demand_curve,lend_to_pool,open_margin_tra
     (receiver, _) = lend_to_pool()
     (loan_id, trader, loan_token_sent, leverage_amount) = open_margin_trade_position()
     startCollateral = sovryn.getLoan(loan_id).dict()["collateral"]
+    deposit_amount = startCollateral/2
     
     #deposit collateral to add margin to the loan created above
-    RBTC.approve(sovryn, startCollateral/2)
-    sovryn.depositCollateral(loan_id, startCollateral/2)
+    RBTC.approve(sovryn, deposit_amount)
+    tx = sovryn.depositCollateral(loan_id, deposit_amount)
+    
+    #verify the deposit collateral event
+    print(tx.info())
```

Comment:
You don't need to use `print`. It also works just with `tx.info()`

Reply:
thanks. if you've got more hints, go ahead ;)